reporter/html.js

```diff
@@ -460,6 +460,18 @@ function storeFixture() {
 	}
 }
 
+function appendFilteredTest() {
+	var testId = QUnit.config.testId;
+	if ( testId && testId.length > 0 ) {
+		return "<div id='qunit-filteredTest'>Running filtered test: " + testId.join(", ") +
```

Comment:
One of the reasons this issue was created was because many people who are starting out on this framework wouldn't think to check the URL, so in terms of UX, I think including the test IDs here wouldn't be redundant.